[azure-storage-*] Update Azure SDK Jun Release

azure-sdk opened the pull request:
Update vcpkg ports for Azure SDK release. This release may contain multiple ports.

azure-sdk committed:
[azure-storage-common-cpp] Update to 12.0.0
## 12.0.0 (2021-06-08)

### Other Changes and Improvements

- Fixed a filename encoding issue.

azure-sdk committed:
[azure-storage-blobs-cpp] Update to 12.0.0
## 12.0.0 (2021-06-08)

### Other Changes and Improvements

- Added and updated some samples.
- Fixed a read consistency issue.

azure-sdk committed:
[azure-storage-files-datalake-cpp] Update to 12.0.0
## 12.0.0 (2021-06-08)

### Breaking Changes

- Renamed `ContentLength` in `FlushFileResult` to `FileSize`.

### Other Changes and Improvements

- Removed `IfUnmodifiedSince` from access conditions of setting filesystem metadata operation.
- Updated some samples.
- Fixed a read consistency issue.

azure-sdk committed:
[azure-storage-files-shares-cpp] Update to 12.0.0
## 12.0.0 (2021-06-08)

### Breaking Changes

- Renamed `ContentLength` in `FileItemDetails` to `FileSize`.

### Other Changes and Improvements

- Updated some samples.
- Fixed a read consistency issue.

ras0219-msft commented:
/azp run

azure-sdk committed:
[azure-security-keyvault-common-cpp] Update to 4.0.0-beta.3
## 4.0.0-beta.3 (2021-06-08)

No breaking changes or new features added. Includes only implementation enhancements.

NancyLi1013 commented:
@ahsonkhan 

Is there anything else that you want to do on your sides now? Since I noticed that you pushed some changes for other ports.

azure-sdk committed:
[azure-security-keyvault-keys-cpp] Update to 4.0.0-beta.3
## 4.0.0-beta.3 (2021-06-08)

### Breaking Changes

- Updated `MaxPageResults` type to `int32_t`, from `uint32_t`, affecting:
  - `GetDeletedKeysOptions()`.
  - `GetPropertiesOfKeysOptions()`.
  - `GetPropertiesOfKeyVersionsOptions()`.
- Updated `CreateRsaKeyOptions::KeySize` type from `uint64_t` to `int64_t`.
- Updated `CreateRsaKeyOptions::PublicExponent` type from `uint64_t` to `int64_t`.
- Updated `CreateOctKeyOptions::KeySize` type from `uint64_t` to `int64_t`.

vhvb1989 commented:
Also adding Azure Security Key Vault Keys 4.0.0-beta.3

NancyLi1013 commented:
Thanks for your information @vhvb1989. Please ping me if this PR is ready for review.

Thanks.

vhvb1989 commented:
@NancyLi1013  yes, please. PR is ready to go

ahsonkhan commented:
@NancyLi1013 and @ras0219-msft can we merge this PR to let customers start adopting and using the Azure storage GA package?

NancyLi1013 commented:
LGTM, thanks for updating these ports.

strega-nil-ms commented:
Thanks azure SDK team :)
